tests/access_chains/array_length_from_access_chain.spv.yaml

```diff
@@ -115,6 +115,7 @@ all_descriptor_bindings:
     block: *bv2 # 
     array: { dims_count: 0, dims: [] }
     accessed: 1
+    access_flags: 0x00000000 # NONE
```

Comment:
last thing left in PR is figure out why this is not consistent with `accessed`

Reply:
this is 

```
         %13 = OpAccessChain %_ptr_StorageBuffer__struct_3 %2
         %14 = OpArrayLength %uint %13 1
                     OpReturn
                     OpFunctionEn
```

and from looking at spec, this is not considered a `read`, just an access